actors/builtin/market/market_actor.go

```diff
@@ -680,7 +680,7 @@ func validateDealCanActivate(proposal *DealProposal, minerAddr addr.Address, sec
 	return nil
 }
 
-func validateDeal(rt Runtime, deal ClientDealProposal, baselinePower, networkQAPower abi.StoragePower) {
+func validateDeal(rt Runtime, deal ClientDealProposal, baselinePower, networkRawPower, networkQAPower abi.StoragePower) {
```

Comment:
nit: would be slightly better to rearrange inputs so that the "raw, qa, baseline" order is the same everywhere to prevent swapping mistakes.  But if there are a lot of changes necessary forget about it.

Reply:
Thanks, I had intended that